### ISSUE TITLE
Integrate open-pip

### ISSUE THREAD

**Issue by albinekb** [fabiospampinato/watch#9]
Hey! Awesome project, i like it 😎 

What do you think of using https://github.com/albinekb/open-pip-cli with this? What's the best way I could support streaming from `watch`?

`$ watch "Rick and morty" | open-pip` 
`$ watch "Rick and morty" | open VLC.app`

The cli would just spit out the streaming URL to stdout, making it possible to use different apps to view the video.

maybe a flag to make it spit out the URL or use a different player? Just thinking.. 👍  any ideas?

**Comment by fabiospampinato** [comment]
It's already possible to stream to a variety of apps, you can see the full list [here](https://github.com/fabiospampinato/webtorrent-cli/tree/iina-support) under the "Options (streaming):" section. My fork of `webtorrent-cli` also supports [IINA](https://github.com/lhc70000/iina), which can open PIP already. 

I guess there's no easy way to integrate `watch` with `open-pip` at the moment.

I like the idea of a flag for outputting the url, but in that case I think we would need to spawn WebTorrent in a detached manner, basically you'll have to kill it manually afterwards.

Maybe a `--pip` flag would be more pratical, allowing for:
```bash
$ watch "Rick and morty" -- --pip
```

I'll see what I can do, thanks for the suggestion.

**Comment by albinekb** [comment]
I've used IINA, it's cool, maybe I'll PR there to create an option to launch it straight into PiP mode, thats probably a way better solution since it has lots more features than my simple CLI.

**Comment by fabiospampinato** [comment]
That would be great.

**Comment by fabiospampinato** [comment]
@albinekb any news on this?

**Comment by albinekb** [comment]
@fabiospampinato nope, been super busy. Not watched anything since last being here 😄 

I think it would be cool if we had an option to just get a http-stream or something to the video file? Then you could play it anywhere 👍
